packaging/scripts/lib/assert-cisco-signature.sh

```
@@ -0,0 +1,104 @@
+# Copyright 2026 Cisco Systems, Inc. and its affiliates
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)